wormchain/x/wormhole/client/cli/genesis.go

```diff
@@ -427,7 +428,7 @@ func CmdSignAddress() *cobra.Command {
 				return err
 			}
 			addr := info.GetAddress()
-			addrHash := crypto.Keccak256Hash(addr)
+			addrHash := crypto.Keccak256Hash(append(wormholesdk.SignedWormchainAddressPrefix, addr...))
```

Comment:
the `...` seems unnecessary? 

```suggestion
			addrHash := crypto.Keccak256Hash(wormholesdk.SignedWormchainAddressPrefix, addr)
```

Reply:
The `...` is actually necessary because `append` is a [variadic function](https://pkg.go.dev/builtin#append).  Dropping the `...` will give a compiler error:

```
cannot use addr (variable of type []byte) as type byte in argument to append
```

See https://go.dev/play/p/3E9zgqrLGG3.

Yes, go is dumb.

Reply:
just leave out the append?

Reply:
done